### PR TITLE
[RM Ch3] Section titles for HW Acceleration

### LINKED ISSUE #2051: [RM Ch03/04] HW Acceleration abstraction
Need to define an abstract model of acceleration and map to some concrete acceleration technologies

### PR DESCRIPTION
**Pull request by petorre**
Fixes #2051 
As per #2051 , RM calls and October vF2F RM session. @TFredberg , @qabpea , @walterkozlowski and I are preparing more PRs with text.

### CODE REVIEW

**Review by walterkozlowski — APPROVED** [review]

**Review by sukhdevkapur — APPROVED** [review]

**Comment by walterkozlowski** [comment]
@pgoyal01, @TFredberg - please review and if you are OK, approve

File: doc/ref_model/chapters/chapter03.md [hunk 1]
```diff
@@ -265,3 +277,15 @@ The following diagram presents an example of the realization of the reference mo
...
+## 3.8 Hardware Acceleration Abstraction
+
+<a name="3.8.1"></a>
+### 3.8.1 Types of accelerators
+
+<a name="3.8.2"></a>
+### 3.8.2 Levels of accelerator consumption
+
+<a name="3.8.3"></a>
+### 3.8.3 Decoupling applications from infrastructure with Hardware Acceleration
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter03.md:291]:
```suggestion
### 3.8.3 Off-loading applications from infrastructure with Hardware Acceleration
```

**Comment by pgoyal01** [comment]
@walterkozlowski Should the "3.8.1 Types of accelerators" or a comment below the title provide guidance that for each type of accelerator, pros and cons as well as application that can benefit should be listed.

File: doc/ref_model/chapters/chapter03.md (hunk 1 above)
Comment (petorre) [reply doc/ref_model/chapters/chapter03.md:291]: Thanks @pgoyal01 . It is meant as "Decoupling" per [Cloud Platform Abstraction Related Terminology](https://github.com/cntt-n/CNTT/blob/master/doc/common/glossary.md#1.6), with levels of decoupling described in text similar to that was before in ex Appendix A.

**Comment by walterkozlowski** [comment]
Agree with @pgoyal01 that in 3.8.1 "Types of of accelerators" should speak about recommended usage, pros and cons of each type

**Review by pgoyal01 — APPROVED** [review]

**Pull request merged**